dumpling/export/config.go

```diff
@@ -593,6 +593,10 @@ func ParseCompressType(compressType string) (storage.CompressType, error) {
 		return storage.NoCompression, nil
 	case "gzip", "gz":
 		return storage.Gzip, nil
+	case "snappy":
+		return storage.Snappy, nil
+	case "zst":
```

Comment:
Do you mean this? It's different with the description in comment
```suggestion
	case "zstd":
```
or maybe
```suggestion
	case "zstd","zst":
```